<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FWPeriodic</class>
 <widget class="QWidget" name="FWPeriodic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Periodic Force</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="headerBox" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Periodic Force&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ForceButtonBox" name="forceButtons" native="true">
        <property name="minimumSize">
         <size>
          <width>313</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_9" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Periodic</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="offsetSpinBox"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Direction</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="directionSpinBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Phase</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="waveformComboBox"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Waveform</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QDoubleSpinBox" name="phaseSpinBox"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Offset</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QDoubleSpinBox" name="periodSpinBox"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Period</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Envelope</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0">
         <item>
          <widget class="FGWPeriodic" name="graphWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="EnvelopeWidget" name="envelopeWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>281</width>
                <height>84</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="durationBox">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="DurationWidget" name="durationWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DurationWidget</class>
   <extends>QWidget</extends>
   <header>DurationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ForceButtonBox</class>
   <extends>QWidget</extends>
   <header>ForceButtonBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EnvelopeWidget</class>
   <extends>QWidget</extends>
   <header>EnvelopeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FGWPeriodic</class>
   <extends>QWidget</extends>
   <header>FGWPeriodic.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
